msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md

```diff
@@ -220,6 +227,234 @@ POST /v3/conversations/{conversationId}/activities
+{
+    content = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+}
+```
+
+# [JavaScript](#tab/javascript)
+
+Not available
+
+# [JSON](#tab/json)
+
+```http
+GET /v1/meetings/{meetingId}
+```
+
+---
+
+The JSON response body for Meeting Details API is as follows:
+
+```json
+{ 
+   "details": { 
+        "id": "meeting ID", 
+        "msGraphResourceId": "", 
+        "scheduledStartTime": "2020-08-21T02:30:00+00:00", 
+        "scheduledEndTime": "2020-08-21T03:00:00+00:00", 
+        "joinUrl": "https://teams.microsoft.com/l/xx", 
+        "title": "All Hands", 
+        "type": "Scheduled" 
+    }, 
+    "conversation": { 
+            "isGroup": true, 
+            “conversationType”: “groupchat”, 
+            "id": "meeting chat ID" 
+    }, 
+    "organizer": { 
+        "id": "<organizer user ID>", 
+        "aadObjectId": "<AAD ID>", 
+        "tenantId": "<Tenant ID>" 
+    }
+} 
+```
+
+## Real-time Teams meeting events
+
+The user can receive real-time meeting events. As soon as any app is associated with a meeting, the actual meeting start and meeting end time are shared with the bot.
+
+Actual start and end time of a meeting are different from the scheduled start and end time. The meeting details API and real-time start and end time meeting events are made available.
+
+There are two major categories of events that are supported:
+
+* Meeting start and end event
+* Participant update event
+
+### Example of meeting start event payload
+
+The following code provides an example of meeting start event payload:
+
+```json
+{
+    "name": "Microsoft/MeetingStart",
+    "type": "event",
+    "timestamp": "2021-04-29T16:10:41.1252256Z",
+    "id": "1619712641110",
+    "channelId": "msteams",
+    "serviceUrl": "https://canary.botapi.skype.com/amer/",
+    "from": {
+        "id": "29:1EbGIDZ3fAuWANLBIoaD4a67K1wHcbOJUzxz068_4WfmoJ7c8LHM_q1w3ALSVWlTAyQRBsV9y7iBGPTzFNfzYVg",
+        "name": "",
+        "aadObjectId": "f30ba569-abef-4e97-8762-35f85cbae706"
+    },
+    "conversation": {
+        "isGroup": true,
+        "tenantId": "e15762ef-a8d8-416b-871c-25516354f1fe",
+        "id": "19:meeting_NTM3ZDJjOTUtZGRhOS00MzYxLTk5NDAtMzY4M2IzZWFjZGE1@thread.v2"
+    },
+    "recipient": {
+        "id": "28:e23cded7-1d12-456c-a93d-15e166a83e10",
+        "name": "Echobot1_WorkMachine2"
+    },
+    "entities": [
+        {
+            "locale": "en-US",
+            "country": "US",
+            "type": "clientInfo"
+        }
+    ],
+    "channelData": {
+        "tenant": {
+            "id": "e15762ef-a8d8-416b-871c-25516354f1fe"
+        },
+        "source": null,
+        "meeting": {
+            "id": "MCMxOTptZWV0aW5nX05UTTNaREpqT1RVdFpHUmhPUzAwTXpZeExUazVOREF0TXpZNE0ySXpaV0ZqWkdFMUB0aHJlYWQudjIjMA=="
+        }
+    },
+    "value": {
+        "MeetingType": "Scheduled",
+        "Title": "Meeting Start/End Event in Canary",
+        "Id":
+"19:meeting_NTM3ZDJjOTUtZGRhOS00MzYxLTk5NDAtMzY4M2IzZWFjZGE1@thread.v2",
```

Comment:
Indent properly

Reply:
Can you please replace the start event with the payload below? This is what I shared in the doc as well @SurbhiJainMSFT 
![image](https://user-images.githubusercontent.com/69164443/119208484-33dd6300-ba57-11eb-913a-e94f2e6aba7e.png)

https://microsoft-my.sharepoint-df.com/:w:/p/rarang/EfIWi3sOGAhLvrjF1gf1ifcBZiSNrrQ4YGqb3_8UqsnTWA?e=8hzRjU